dev/io.openliberty.microprofile.openapi.internal.common/resources/io/openliberty/microprofile/openapi/internal/resources/OpenAPI.nlsprops

```diff
@@ -130,3 +130,15 @@ OPEN_API_DOC_PATH_INVALID_CWWKO1676E.useraction=Ensure the OpenAPI Path contains
 OPEN_API_PATH_SEGMENT_INVALID_CWWKO1677E=CWWKO1677E: Path segment "/." or "/.." is invalid for OpenAPI endpoints.
 OPEN_API_PATH_SEGMENT_INVALID_CWWKO1677E.explanation=OpenAPI path segment "/." or "/.." is invalid.
 OPEN_API_PATH_SEGMENT_INVALID_CWWKO1677E.useraction=Remove any path "/." or "/.." segment from the path.
+
+OPEN_API_SLASH_IN_APPLICATION_CWWKO1678W=CWWKO1678W: Path segment "/" is in {0} invalid for applications.
+OPEN_API_SLASH_IN_APPLICATION_CWWKO1678W.explanation=explanation: An application cannot name cannot include a "/" as "/" is used to separate the application from its component modules.
+OPEN_API_SLASH_IN_APPLICATION_CWWKO1678W.useraction=useraction: Remove the "/" from the includeApplication or excludeApplication element in your server.xml, or convert that element to includeModule or excludeModule. 
+
+OPEN_API_SLASH_IN_MODULE_CWWKO1679W=CWWKO1679W: Path segment "/" is required for modules but was absent in {0}.
+OPEN_API_SLASH_IN_MODULE_CWWKO1679W.explanation=explanation: An module must be in the format <application-name>/<module-name>, therefore a module name without a "/" is invalid.
+OPEN_API_SLASH_IN_MODULE_CWWKO1679W.useraction=useraction: Correct the includeModule or excludeModule element in your server.xml, or convert that element to includeApplication or excludeApplication. 
```

Comment:
Message guidelines say that that "unrecoverable configuration errors" are errors while "configuration issues" are warnings.

My initial instinct was that these should be errors, but reading the guidelines I think you're correct that they're warnings because they don't prevent the server from running.